python/ray/_private/state.py

```diff
@@ -762,13 +762,6 @@ def _available_resources_per_node(self):
             node_id = ray._private.utils.binary_to_hex(message.node_id)
             available_resources_by_id[node_id] = dynamic_resources
 
-        # Update nodes in cluster.
-        node_ids = self._live_node_ids()
```

Comment:
Why are they deleted?

Reply:
I think there is no need to do the filtering again since GCS only returns available resources of live nodes.